cmd/feedback/feedback.go

```diff
@@ -68,8 +68,9 @@ const (
 
 // Supported survey names
 const (
-	SlackCLIFeedback      = "slack-cli"
-	SlackPlatformFeedback = "platform-improvements"
+	SlackCLIFeedback                = "slack-cli"
+	SlackPlatformFeedback           = "slack-platform"
+	SlackPlatformFeedbackDeprecated = "platform-improvements" // DEPRECATED(semver:major)
```

Comment:
note: Added a `DEPRECATED(semver:X)` comment to make this easier to find in the future. We are a little consistent with our deprecation comments - `TODO(semver:major)` and others also exist. Preference?

Reply:
Thanks for both this comment and calling out the formatting!

IIRC we are using `TODO(semver:major)` more often, but this is often for internal breaking changes AFAICT.

I think `DEPRECATED` is meaningful here since it captures the ongoing support with an existing alternative option, but IMO the `semver:major` part is the most important! This is what I would use in searching for these 😉

Reply:
Also I learned in review of #71 that `go` has a convention around making machine-readable comments that I'm interested in exploring:

> ...this comment directives are line comments that start with `//go:`, with no space between the `//` and the `go:`.

https://go.dev/wiki/Comments#directives

Using a similar pattern might be interesting for these more "conventional" comments, but I don't believe we have explicit [notes](https://github.com/slackapi/slack-cli/blob/main/.github/MAINTAINERS_GUIDE.md) on how these should be used overall...

Reply:
One addition to this comment might also note the deprecated suggestions? 📚

Edit: I realize you make a similar comment a few lines down - no blocker here!

```suggestion
	SlackCLIFeedback                = "slack-cli"
	SlackPlatformFeedback           = "slack-platform"
	SlackPlatformFeedbackDeprecated = "platform-improvements" // DEPRECATED(semver:major) - Replaced with the "slack-platform" option
```

Reply:
The `//go:` directive is interesting. It led me down a path to find the use of the [`// DEPRECATED:` comment in Golang](https://cs.opensource.google/search?q=Deprecated:%20language:go&ss=go%2Fgo). So, perhaps we can start to use that convention to signal deprecated code. I definitely agree that the `(semver:major)` scope is most important though!